### DISCUSSION TITLE
clolumn微服务模式启动，生成项目后配置文件中，服务名错误，导致注册nacos时与该服务名字不一致

### DISCUSSION THREAD

**Discussion by chenhaonanX** [zuihou/zuihou-admin-cloud#362] [企业版]
![c1cc1ffd65f7e9b0f4a5162f6510834](https://github.com/user-attachments/assets/1f79680b-9630-4bd5-99d6-36c2ef7ae9c9)

**Comment by zuihou** [comment]
已修复： https://github.com/dromara/lamp-cloud/commit/f8740fe9e120bddb33914f418769542d61507b5b
